avocado/core/test.py

```diff
@@ -148,7 +152,7 @@ class Test(unittest.TestCase):
     timeout = None
 
     def __init__(self, methodName='test', name=None, params=None,
-                 base_logdir=None, job=None, runner_queue=None):
+                 base_logdir=None, job=None, runner_queue=None, loader=None):
```

Comment:
This is a change I dislike.  IIUC, the `loader` parameter is needed because that information would then be needed later to build the matrix of test loaders x tests found, right?

We have to redesign this in such a way that this data is kept outside the `avocado.core.test.Test` class.